### ISSUE TITLE
Figure out a way around GitHub not sending submodules in tarballs

### ISSUE THREAD

**Issue by jipiboily** [rainforestapp/fourchette#16]
I started to use the GitHub API to get a temporary link to a tarball of the branch we want a Heroku fork for. The downside is that if you use submodules, you're screwed as it's not part of the tarball (which makes sense I think).

Possible solutions:
- do a clone, make a tarball of it and publicly available for Heroku for a few seconds at some random an long URL + with Heroku token or something like that
  or
- have the same thing, but as a different and optional app, a github-to-tarball as a service sort of thing.
  or
- have Heroku support Git + submodules while cloning it.

The fine folks @ Heroku are aware of that, waiting on them but I am not sure I'll wait for them before implementing something better.
